src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -617,6 +629,31 @@
     text: az sql midb show -g mygroup --mi myinstance -n mymanageddb
 """
 
+helps['sql midb retention-policy'] = """
+type: group
+short-summary: Manage SQL managed instance database backup retention policy.
+"""
+
+helps['sql midb retention-policy update'] = """
+type: command
+short-summary: Update short term retention for automated backups on a single database.
```

Comment:
Other clients (PS) have the following as the documentation for this api, do we want consistency?

> Sets a backup short term retention policy.

Reply:
You are right, will change update to set.